dashboard/utils.py

```diff
@@ -277,6 +282,47 @@ def count_courses_passed(self):
                 passed_courses.add(self.edx_key_course_map[course_id])
         return len(passed_courses)
 
+    def get_pearson_exam_status(self):  # pylint: disable=too-many-return-statements
+        """
+        Get the pearson exam status for the user / program combo
+        """
+        if self.program.exam_series_code is None:
+            return ""
+        else:
+            user = self.user
+            try:
+                exam_profile = ExamProfile.objects.get(profile=user.profile)
+            except ExamProfile.DoesNotExist:
+                return ExamProfile.PROFILE_ABSENT
+
+            if (exam_profile.status == ExamProfile.PROFILE_PENDING or
```

Comment:
Could be simplified to `exam_profiles.status in (ExamProfile.PROFILE_PENDING, ExamProfile.PROFILE_IN_PROGRESS)`